.github/workflows/triage_handle_new_comments.yml

```diff
@@ -0,0 +1,31 @@
+name: 'Handle Issue/PR Comment Workflow'
+on:
+  # makes this workflow reusable
```

Comment:
yes from my understanding it will pull down the cypress code in order to run the script within the context of that repo.  I have not tested this aspect out yet and it isn't used.  A future PR will happen that will be for how other repos under the cypress org utilize this workflow.

Reply:
This repo is pretty massive. I am wondering if this code would be better suited for https://github.com/cypress-io/release-automations. Is there issues with re-usable workflows from a private repo?

Reply:
I will need to investigate that approach.  I think it's a good idea but I have not tested that.

Reply:
I've moved the script to the release-automations repo